code/modules/chemistry/tools/extinguisher.dm

```diff
@@ -175,7 +175,7 @@
 		else if( usr.buckled && !usr.buckled.anchored )
 			var/wooshdir = get_dir( target, user )
 			SPAWN_DBG(0)
-				for( var/i = 1, (usr && usr.buckled && !usr.buckled.anchored && i <= rand(3,5)), i++ )
+				for( var/i = 1, (usr?.buckled && !usr.buckled.anchored && i <= rand(3,5)), i++ )
```

Comment:
wtffff

Reply:
yea it's kind of weird...from what I remember in another runtime patch they shoot 3-5 sprays in 1 of 3 turfs chosen at random

This can mean all 5 go to 1 lane, or only 3 spawn but each lanes gets one. Causes it to have the random spray patterns.